ExternalTest/rr_sim_tests/docker/Dockerfile

```
@@ -33,5 +35,5 @@ RUN apt update && apt install -y python3-pip ros-foxy-tf-transformations && pip3
 RUN cp ${ROS2_WS}/src/rr_sim_tests/docker/fastdds_setup.sh /
 RUN cp ${ROS2_WS}/src/rr_sim_tests/docker/fastdds_config.xml /
 
-CMD /bin/bash -c ". ${ROS2_WS}/install/setup.bash && source fastdds_setup.sh && ros2 run rr_sim_tests test_random_spawn --ros-args -p robot_name:=${ROBOT_NAME}"
+CMD /bin/bash -c ". ${ROS2_WS}/install/setup.bash && source fastdds_setup.sh && ros2 run rr_sim_tests test_random_spawn --ros-args -p robot_name_prefix:=${ROBOT_NAME} -p start_index:=${START_INDEX} -p robot_num:=${ROBOT_NUM}"
```

```suggestion
CMD /bin/bash -c ". ${ROS2_WS}/install/setup.bash && source fastdds_setup.sh && ros2 run rr_sim_tests test_random_spawn --ros-args -p robot_name_prefix:=${ROBOT_NAME_PREFIX} -p start_index:=${START_INDEX} -p robot_num:=${ROBOT_NUM}"
```
I think It supposed to be the prefix?

---

ah yes. thank you.